### ISSUE TITLE
Issue with reading PPA data using Historian Data Viewer tool

### ISSUE THREAD

**Issue by xtrememenace** [GridProtectionAlliance/openPDC#106]
I used the PPA historian (historian output is saved as PPA1) to log the PMU data to the archive folder where openPDC is installed on system A. The data from the PMU is stored at 120 fps. Based on the .d formatted files stored in the archive folder, I have 165GB total size. I can browse through this data using the historian data viewer.exe tool inside the openPDC folder on system A but if I copy the archive folder and point it to another historian data viewer.exe installed on system B, I can't see any data. I have changed the tool's archive path on system B to point it to the correct location, but I don't see any data. 
![capture](https://user-images.githubusercontent.com/2134842/37793539-9f8f0184-2dd4-11e8-86a4-99e7f3d17cf9.PNG)
.
.
.
![capture2](https://user-images.githubusercontent.com/2134842/37793597-c80e3bde-2dd4-11e8-8926-d2bd7381e7c2.PNG)
Do I need to copy any more information to the second openPDC installation on system B to see my archived data from system A or is there any other standalone tool that I can use to view the archived data?

**Comment by ritchiecarroll** [comment]
I notice some files are ppa1 and others are ppa?

**Comment by xtrememenace** [comment]
Yes. I am using PPA1 as the historian output and it is same as the original historian PPA but with a different name. After installing openPDC on system A, I created a new historian named PPA1 but with same settings as PPA. PPA historian is not enabled. openPDC historian output screenshot from system A:
![capture](https://user-images.githubusercontent.com/2134842/37799141-6c9dc0d4-2de4-11e8-8dde-c0d0a20db9b5.PNG)
Historian Data Viewer on system B (Note, the archive I'm reading in this tool is PPA1 data copied from system A):
![capture3](https://user-images.githubusercontent.com/2134842/37799196-97d85d72-2de4-11e8-9690-da14697a3e20.PNG)
